### PR TITLE
Document Lozenge Usage specs

### PR DESCRIPTION
**Pull request by fraincs**
Issue: 

## Summary

Lozenge had no best practices and accessibility sections, this PR aims at adding more information to better guide devs and designers in how to use this component.

## What I did

Reworded some usage sections, added a best practices, related and accessibility section.

## How to test

 visit https://deploy-preview-956--sg-storybook.netlify.app/?path=/docs/lozenge--default-story

### CODE REVIEW

**Comment by fraincs** [comment]
Anatomy has not been added, this should be done.

File: packages/components/src/lozenge/docs/Lozenge.stories.mdx [hunk 1]
```diff
@@ -12,18 +12,35 @@ import { Text } from "@components/typography";
...
+<Tagline>Lozenges are used to highlight an item's status, for quick recognition or to give additional information such as highlighting a new item.</Tagline>
 
 <ComponentInfo
     usage={"import { Lozenge } from \"@sharegate/orbit-ui\";"}
     githubPath="/packages/components/src/lozenge/src"
 />
 
+## Best Practices
+
+- Use to highlight an element in a list.
```
Comment (LaurentDario) [inline packages/components/src/lozenge/docs/Lozenge.stories.mdx:24]: I'm wondering if these should be listed under "Usage" bellow as a list of scenarios we can use the Lozenge for.

File: packages/components/src/lozenge/docs/Lozenge.stories.mdx [hunk 2]
```diff
@@ -12,18 +12,35 @@ import { Text } from "@components/typography";
...
+- Use to bring attention to a particular element that requires the user’s attention, such as an expiring license or a problematic situation.
```
Comment (LaurentDario) [inline packages/components/src/lozenge/docs/Lozenge.stories.mdx:25]: The other scenario we use it for is to highlight something new, not sure it's relevant to add it as an example but if you think it is...

File: packages/components/src/lozenge/docs/Lozenge.stories.mdx [hunk 3]
```diff
@@ -12,18 +12,35 @@ import { Text } from "@components/typography";
...
+- Keep the copy short, ideally only 1 word or 2 short words. If you need to display more text, you should consider using another component such as a [message](?path=/docs/message--default-story) or an [alert](?path=/docs/alert--default-story).
+- Don't render this element interactive in any way, this includes making it clickable.
```
Comment (LaurentDario) [inline packages/components/src/lozenge/docs/Lozenge.stories.mdx:27]: One scenario that I could see happening a lot is using a `Tooltip` to add content to a Lozenge. Should we specify that it shouldn't be used with a Lozenge?

**Review by LaurentDario — APPROVED** [review]
Just a few comments on copy!

File: packages/components/src/lozenge/docs/Lozenge.stories.mdx (hunk 1 above)
Comment (fraincs) [reply packages/components/src/lozenge/docs/Lozenge.stories.mdx:24]: Usage is used in a how to use the component in code, that's why best practices wouldn't fit under usage section.

File: packages/components/src/lozenge/docs/Lozenge.stories.mdx (hunk 3 above)
Comment (fraincs) [reply packages/components/src/lozenge/docs/Lozenge.stories.mdx:27]: > Don't render this element interactive in any way, this includes making it clickable

pretty much sums it up but adding that specification is not out of scope

Comment (LaurentDario) [reply packages/components/src/lozenge/docs/Lozenge.stories.mdx:27]: It's just that technically, adding a tooltip doesn't make it "clickable" per say. But are we ok with a tooltip on hover of a Lozenge?

Comment (fraincs) [reply packages/components/src/lozenge/docs/Lozenge.stories.mdx:27]: Don't render this element interactive was the hint / interactive includes hovering an element and reacting to it.

**Pull request merged**